apis/Google.Cloud.Compute.V1/Google.Cloud.Compute.V1.IntegrationTests/AcceptanceTest.cs

```diff
@@ -0,0 +1,125 @@
+﻿// Copyright 2021 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     https://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+using System;
+using System.Threading;
+using Grpc.Core;
+using Xunit;
+using Xunit.Abstractions;
+
+namespace Google.Cloud.Compute.V1.IntegrationTests
+{
+    /// <summary>
+    /// Miscellaneous acceptance tests for the Compute API.
+    /// </summary>
+    [Collection(nameof(ComputeFixture))]
+    public sealed class AcceptanceTest
+    {
+        private readonly ITestOutputHelper _output;
+        private readonly ComputeFixture _fixture;
+
+        public AcceptanceTest(ComputeFixture fixture, ITestOutputHelper output) =>
+            (_fixture, _output) = (fixture, output);
+
+        [Fact]
+        public void CreateDeleteIPAddress()
```

Comment:
I would write individual tests with proper resuource initialization and cleanup after.
We wouldn't need to write so much to the output then.
Basically, this is a big departure from how we write integration tests in this repo and I would prefrer if we stick to one way of writing tests. I think these were going to be worked on more, so maybe something to take into account.

Reply:
We've got a *few* tests that are like this - particularly in Storage, for one access control feature. (Can't remember the name now.)

I did consider doing resource clean up in the disposal part - listing all IP addresses either specifying a filter (if there's such a thing as a "name starts with" filter) or deleting everything starting with "testaddr". Or possibly generating a per-test prefix. Thoughts on that?

Reply:
You'd probably need a per-test prefix, there's a chance that Windows and Ubuntu are running at the same time and one tramples on the other's IPs.

Reply:
Yup - it'll be part of the test fixture.